README.md

```diff
@@ -125,6 +125,19 @@ fetch('http://httpbin.org/post', { method: 'POST', body: stream })
 		console.log(json);
 	});
 
+// post with JavaScript object
+
+var body = { a: 1 };
+fetch('http://httpbin.org/post', { 
+        method: 'POST',
+        body:    JSON.stringify(body),
+        headers: { 'Content-Type': 'application/json' },
+    }).then(function(res) {
+        return res.json();
+    }).then(function(json) {
+        console.log(json);
+    });
```

Comment:
I will squash it, you can just fix the formatting.

Though I do wonder how often people use this when `form-data` is available.